docker/texlive-profile.txt

```diff
@@ -0,0 +1,10 @@
+selected_scheme scheme-minimal
+TEXDIR /usr/local/texlive
+TEXMFCONFIG ~/.texlive/texmf-config
+TEXMFHOME ~/texmf
+TEXMFLOCAL /usr/local/texlive/texmf-local
+TEXMFSYSCONFIG /usr/local/texlive/texmf-config
+TEXMFSYSVAR /usr/local/texlive/texmf-var
+TEXMFVAR ~/.texlive/texmf-var
+option_doc 0
+option_src 0
```

Comment:
Is it ok that we distribute this without docs and source? Does the TeXlive license allow this?

Reply:
These variables control whether sources and documentation for the installed packages is kept.

I don't see any licensing issues; our project license only concerns the `Dockerfile`, not the software installed with it.